### PR TITLE
Add Quickstart section to README.org

### PR DESCRIPTION
**Pull request by codevbus**
Updated with a quick start section

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -1,4 +1,39 @@
 * Emacs overlay for Nixpkgs
+** Quickstart
+To get up and running quickly, add the following lines to your =/etc/nixos/configuration.nix=:
+
+#+BEGIN_SRC nix
+{config, pkgs, callPackage, ... }:
+{
+...
+
+services.emacs.package = pkgs.emacsUnstable
```
Comment (fritz-k) [inline README.org:10]:
```suggestion
services.emacs.package = pkgs.emacsUnstable;
```

**Comment by Mic92** [comment]
I could merge this now if it was rebased.

**Comment by codevbus** [comment]
@Mic92 - I'll rebase it, thanks for following up on the long tail!

File: README.org [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+
+services.emacs.package = pkgs.emacsUnstable;
+
+nixpkgs.overlays = [
+  (import (builtins.fetchTarball {
+    url = https://github.com/nix-community/emacs-overlay/archive/master.tar.gz;
+  }))
+];
+
+...
```
Comment (talyz) [inline README.org:18]: Nice work! Thanks! Just one suggestion:
```suggestion
# ...

  services.emacs.package = pkgs.emacsUnstable;

  nixpkgs.overlays = [
    (import (builtins.fetchTarball {
      url = https://github.com/nix-community/emacs-overlay/archive/master.tar.gz;
    }))
  ];

# ...
```

**Comment by codevbus** [comment]
@Mic92  - should be g2g, sorry history/commit is a little messy.

**Comment by talyz** [comment]
Thanks!

**Pull request merged**